db/system_keyspace.cc

```diff
@@ -96,6 +96,9 @@ namespace {
             system_keyspace::COMMITLOG_CLEANUPS,
             system_keyspace::SERVICE_LEVELS_V2,
             system_keyspace::VIEW_BUILD_STATUS_V2,
+            system_keyspace::CDC_STREAMS_STATE,
+            system_keyspace::CDC_STREAMS_HISTORY,
+            system_keyspace::CDC_PENDING_STREAMS,
```

Comment:
> cdc: add internal tables for cdc with tablets

How internal are they? Do the drivers access them? Or is the driver protocol unchanged?